src/features/web-compat.js

```diff
@@ -35,10 +35,10 @@ function canShare (data) {
 
 /**
  * Clean data before sending to the Android side
- * @returns {ShareRequestData}
+ * @returns {import('../types/web-compat.js').WebShareParams}
```

Comment:
I wonder if we could alias this to the feature class itself?
Something like:`WebCompat.Messages.WebShareParams`

I know this isn't core to the design itself just would be nice to not require feature authors to alias into the type dir.

Reply:
+1 this feels confusing because the you're supposed to think of what exactly would be generated.
I guess for new features we can auto-generate the whole boilerplate ContentFeature subclass, but I like Jonathan's suggestion more

Reply:
Removed - none of this is needed in the new design!